src/imitation/scripts/common/wb.py

```diff
@@ -0,0 +1,64 @@
+"""Weights & Biases configuration elements for scripts."""
+
+import logging
+from typing import Any, Mapping, Optional
+
+import sacred
+
+wandb_ingredient = sacred.Ingredient("common.wandb")
+logger = logging.getLogger(__name__)
+
+
+@wandb_ingredient.config
+def wandb_config():
+    """Other user can overwrite this function to customize their wandb.init() call."""
+    wandb_tag = None  # User-specified tag for this run
+    wandb_name_prefix = ""  # User-specified prefix for the run name
+    wandb_kwargs = dict(
+        project="imitation",
+        monitor_gym=False,
+        save_code=False,
+    )  # Other kwargs to pass to wandb.init()
+
+    locals()
+
+
+@wandb_ingredient.capture
+def wandb_init(
+    _run,
+    wandb_name_prefix: str,
+    wandb_tag: Optional[str],
+    wandb_kwargs: Mapping[str, Any],
+    log_dir: str,
+) -> None:
+    """Putting everything together to get the W&B kwargs for wandb.init().
+
+    Args:
+        wandb_name_prefix (str): User-specified prefix for wandb run name.
```

Comment:
I see. I included types because Copilot autogenerated docstrings for me. 
I'm aware that you use `darglint` to check those docstrings. I might also add it to RFT repo for docstring check.